CONTRIBUTING.md

```diff
@@ -5,32 +5,34 @@
 This project uses a set of linters to ensure good code quality.
 In order to make proper use of those linters inside an IDE, the following configuration is required.
 
-Further information can also be found in the [`golangci-lint` documentation](https://golangci-lint.run/usage/integrations/).
+Further information can also be found in
```

Comment:
Once, I had an excellent discussion regarding documentation, line-length, and how to split it. I want to share the outcome, as I think it is an exciting consideration we might want to apply.

1. line length in coding etc., is essential, as too long lines are hard to read - the same somewhat applies to documentation and readmes, too - but there is one difference - word-wrap within the tools handles this quiet well, compared to source code.
2. Adding breaks in between sentences make them harder to rearrange, also if multiple sentences are within one line, make it harder to reorder those sentences.
3. Most documentation tools do not see one line break as a new paragraph and they will print the text fluidly anyways

So what we did/agreed upon in this other project (yes this might be controversial) is for documentation text (readme, docs etc) is following:

1. each sentence is on one line, not two not more
2. we do not care about line length for text for two reasons:
    - to comply with rule 1
    - to spot too long sentences more easily, which might need to be rewritten

This allowed us to use IDE tools to move around sentences easily, detect complicated text structures, and not struggle with where to break :)

As mentioned this is just a discussion from another project I was part, and i think it is a nice consideration, and a different take worth exploring/mentioning

Reply:
I do quite like the one line - one sentence approach as well :)
The question is, can we enforce it through linting? If there's a way to enforce it, I'm all for it! If not, I don't think it's the best idea :)

Reply:
there is a way to enforce it - https://github.com/JoshuaKGoldberg/sentences-per-line - the question is, do we build our docker image for containing this rule, or how do we handle the inclusion of the custom rule :) i will explore this on Monday - worst case would be an own extended docker image in an own repository, containing this custom rule

Reply:
maybe it can also be contributed to markdownlint as a new optional rule

Reply:
https://github.com/DavidAnson/markdownlint/issues/66#issuecomment-1427886697 :D